insights/parsers/virtlogd_conf.py

```diff
@@ -0,0 +1,94 @@
+"""
+VirtlogdConf - file ``/etc/libvirt/virtlogd.conf``
+==================================================
+
+The VirtlogdConf class parses the file ``/etc/libvirt/virtlogd.conf``. The
+virtlogd.conf is in the standard 'ini' format and is read by the base
```

Comment:
Would this be standard `conf` file format instead of `ini` format?

Reply:
:+1: